micro-infra-spring-base/build.gradle

```diff
@@ -31,9 +31,9 @@ dependencies {
     testCompile "org.spockframework:spock-core"
     testRuntime "org.spockframework:spock-spring"
     testCompile "org.springframework:spring-test"
-    testCompile 'org.hamcrest:hamcrest-library:1.3'
+    testCompile 'org.hamcrest:hamcrest-library:1.4-atlassian-1'
```

Comment:
It is not listed here:
https://code.google.com/p/hamcrest/downloads/list

However it looks that some libraries already use it:
https://www.versioneye.com/java/org.hamcrest:hamcrest-core/1.4-atlassian-1